### PR TITLE
[#5928] fix(CLI): Fix columns details command produces no output

### PR DESCRIPTION
**Pull request by Abyss-lord**
### What changes were proposed in this pull request?

Fix columns details command produces no output, when command without --audit option, cli should tell user that command is not support.

### Why are the changes needed?

Fix: #5928 

### Does this PR introduce _any_ user-facing change?

NO

### How was this patch tested?

local test.

### CODE REVIEW

**Comment by Abyss-lord** [comment]
Hi @justinmclean @xunliu @tengqm , could you please review this PR when you have time? I’d really appreciate your feedback.

File: clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java [hunk 1]
```diff
@@ -755,6 +755,8 @@ private void handleColumnCommand() {
       case CommandActions.DETAILS:
         if (line.hasOption(GravitinoOptions.AUDIT)) {
           newColumnAudit(url, ignore, metalake, catalog, schema, table, column).handle();
+        } else {
+          System.out.println(ErrorMessages.UNSUPPORTED_ACTION);
```
Comment (justinmclean) [inline clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java:759]: need a Main.exit(-1) here

Comment (Abyss-lord) [reply clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java]: Hi @justinmclean , I’ve finished updating the code. Please take a look at the PR again when you have time.

**Review by justinmclean — APPROVED** [review]
Thanks for the PR

**Review by xunliu — APPROVED** [review]
@Abyss-lord Thank you for your contributions.
LGTM

**Pull request merged**